schema/deploy/shipping/views.sql

```diff
@@ -225,4 +225,33 @@ create or replace view shipping.observation_with_presence_absence_result_v1 as
       join shipping.presence_absence_result_v1 using (sample)
       order by site, encounter, sample, target;
 
+
+create or replace view shipping.flu_assembly_jobs_v1 as
+
+    select sample.identifier as sfs_uuid,
+           sample.details ->> 'nwgc_id' as nwgc_id,
+           sequence_read_set.urls,
+           target.identifier as target,
+           o1.lineage as target_linked_organism,
+           case (o2.lineage) is not null
+             when true
+             then o2.lineage
+             else o1.lineage
+           end as assembly_job_organism,
+           sequence_read_set.details
+
+      from warehouse.sequence_read_set
+      join warehouse.sample using (sample_id)
+      join warehouse.presence_absence using (sample_id)
+      join warehouse.target using (target_id)
+      join warehouse.organism o1 using (organism_id)
+      left join warehouse.organism o2 on (o2.lineage ~ concat(o1.lineage::text, '.*{1}')::lquery)
```

Comment:
I'm not sure I understand why we want this join condition.

Reply:
Jover and I chatted in person, and this makes sense now: for pan-subtype targets like `Flu_A_pan` which detects H1 and H3, we want to spread out to the more specific lineages for assembly jobs.

It'd be great to add a comment to the query explaining why the `o2` join happens and what the join condition is doing, plus how we use the `o2` lineage in the sequence read set details.